public/app.js

```diff
@@ -0,0 +1,330 @@
+// AUTH CONTROLLER
+var authController = (function() {
+    return {
```

Comment:
I don't JS so much these days, but I believe you can just say:
```javascript
let authController = {
    initApp: function() {...},
    ...
};
```
i.e. no need to create and then call a function

Reply:
My idea was to create the functions and then call them on the global app controller where the logic of the dashboard is. What do you think about that?

Reply:
I think I see what you're doing now. Consider:
```javascript
class AuthController {
    initApp: function() {...},
    ...
}
```
Then yes, in the global controller you can say `AuthController.initApp()`.

Reply:
The AuthController will be undefined because it will not have been executed

Reply:
works for me: https://jsbin.com/davalofebu/edit?js,console

Reply:
Noted your implementation, thanks